Resources/doc/index.rst

```diff
@@ -1,15 +1,167 @@
-=============
-Configuration
-=============
+===========
+Quick start
+===========
 
-First of all you must config which shop you are going to work with: 
+Let's get started. We'll guide you step by step through installing and running ``ONGR sandbox`` integration with ``Magento`` for
+the first time. The first installation should not need more than 1.5 hour.
 
-.. code-block:: yaml
+Step 1: Requirements.
+---------------------
+
+Yes there are a few.
+
+Please check if your development environment in our handbook.
+
+Step 2: Download ONGR
+---------------------
+
+Download the latest release here `archive <https://github.com/ongr-io/ongr-sandbox/releases>`_ and unpack it somewhere under your project directory. Make sure that we have the "Vagrantfile" in the your project root folder.
```

Comment:
When making links, it is advisable to avoid the usage of here. It should probably look something akin to this: Download latest release from the \`archive...